src/onegov/org/models/ticket.py

```diff
@@ -415,6 +396,14 @@ def undecided(self):
 
         return True
 
+    def prepare_delete_ticket(self):
+        for reservation in self.reservations or ():
+            self.session.delete(reservation)
+
+    @property
+    def ticket_deletable(self):
+        return not self.has_future_reservation() and super().ticket_deletable()
```

Comment:
I'm not sure how `super()` interacts with properties, but I think this should still be `super().ticket_deletable` without the call.